node/tests/Unison/Test/Index.hs

```diff
@@ -2,6 +2,7 @@
 {-# Language FlexibleInstances #-}
```

Comment:
If you want to leave it as a type alias (vs converting to a newtype), you shouldn't need an `Arbitrary` instance for `Indentifier`, just an `Arbitrary` for `Series`, and let the pair `Arbitrary` take care of the rest. If not just any pair of series is a valid `Identifier`, I'd make it a `newtype`.